### PR TITLE
Feature/code cleanup

### PR DESCRIPTION
**Pull request by Crow08**
- removed all *- imports
- avoid the use of deprecated API.
- migrate from Date to Instant.
- fixed maven warnings.

### CODE REVIEW

File: neverpile-eureka-authorization/src/test/java/com/neverpile/authorization/service/impl/SimpleMutablePolicyRepositoryTest.java [hunk 1]
```diff
@@ -88,7 +93,7 @@ SimpleMutablePolicyRepository documentService() {
   ObjectMapper objectMapper;
 
 
-  String policyPattern = "{\"validFrom\" : \"2018-01-01\",\"description\": \"%s\",\"default_effect\": \"DENY\",\"rules\": []}";
+  String policyPattern = "{\"validFrom\" : \"2018-01-01T00:00:00.000Z\",\"description\": \"%s\",\"default_effect\": \"DENY\",\"rules\": []}";
```
Comment (Crow08) [inline neverpile-eureka-authorization/src/test/java/com/neverpile/authorization/service/impl/SimpleMutablePolicyRepositoryTest.java]: important change here!

File: neverpile-eureka-hazelcast/src/main/java/com/neverpile/eureka/hazelcast/lock/HazelcastSimpleLockFactory.java [hunk 2]
```diff
@@ -6,22 +6,22 @@
...
 
-import com.hazelcast.cp.CPSubsystem;
+import com.hazelcast.core.HazelcastInstance;
 import com.neverpile.eureka.tx.lock.ClusterLockFactory;
 
 public class HazelcastSimpleLockFactory implements ClusterLockFactory {
   protected static final Logger logger = LoggerFactory.getLogger(HazelcastSimpleLockFactory.class);
 
   @Autowired
-  private CPSubsystem cp;
```
Comment (Crow08) [inline neverpile-eureka-hazelcast/src/main/java/com/neverpile/eureka/hazelcast/lock/HazelcastSimpleLockFactory.java]: cp subsystem cannot be used right now...

File: neverpile-eureka-bridge-storage-cassandra/src/main/java/com/neverpile/eureka/bridge/storage/cassandra/CassandraHealthIndicator.java [hunk 3]
```diff
@@ -48,7 +48,7 @@ public Health health() {
       builder.withDetail("Open Connections", state.getOpenConnections(f));
       builder.withDetail("Trashed Connections", state.getTrashedConnections(f));
 
-      if (!f.getState().equals(Status.UP) && checkSessionState()) {
+      if (!f.getState().equals(Status.UP.getCode()) && checkSessionState()) {
```
Comment (Crow08) [inline neverpile-eureka-bridge-storage-cassandra/src/main/java/com/neverpile/eureka/bridge/storage/cassandra/CassandraHealthIndicator.java]: was this an error? (Comparison between State and String)

Comment (hennejg) [reply neverpile-eureka-bridge-storage-cassandra/src/main/java/com/neverpile/eureka/bridge/storage/cassandra/CassandraHealthIndicator.java]: Yep, it was...

**Comment by hennejg** [comment]
Looks great (and like a lot of work)...

**Pull request merged**